lib/manageiq/util/file_splitter.rb

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env ruby
```

Comment:
I'd rather reuse the manageiq/tools directory than introduce a new util dir.

Reply:
This `util` dir already existed, pretty sure...

https://github.com/ManageIQ/manageiq/tree/d9dd44b/lib/manageiq/util

But also, this is from https://github.com/ManageIQ/manageiq/pull/17549 so I would prefer comments on `file_splitter.rb` would be done there.  This is just making use of it because I need to keep moving forward.

Reply:
Nope, https://github.com/ManageIQ/manageiq?files=1

Reply:
Oops, nvm... Thanks misread this as /util.  This is fine.